sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_models.py

```diff
@@ -53,3 +53,32 @@ class SettingSelector:
     def __init__(self, *, key_filter: str, label_filter: Optional[str] = EMPTY_LABEL):
         self.key_filter = key_filter
         self.label_filter = label_filter
+
+
+class RefreshRegistrations:
+    def __init__(self, key_filter: str, label_filter: Optional[str], refresh_all: Optional[bool]):
+        self.key_filter = key_filter
+        self.label_filter = label_filter
+        self.refresh_all = refresh_all
+        self.etag = None
+
+
+class AzureAppConfigurationRefreshOptions:
+    def __init__(self):
+        self.refresh_interval = 30
```

Comment:
I dont think its a good user experience to have to calculate the refresh interval in seconds, especially when it could be a long time interval too. Is there a Python equivalent for C# [TimeSpan](https://learn.microsoft.com/en-us/dotnet/api/system.timespan?view=net-7.0) that we could use for refresh_interval? If there's no other option, the current name of the property is confusing because its not obvious that its supposed to be in seconds.

Reply:
There isn't. I have been talking with the SDK team about it. We would provide a way for them to provide days/hours/minutes/seconds, but we would just be doing the same calculation back to seconds.

Reply:
@annatisch and better ideas?

Reply:
Let's just leave this as an integer for now. Seconds is generally the default, so the name doesn't need any kind of unit suffix and shouldn't be confusing. If we really want to, we can add support for `Union[int, timedelta]`.
However we can also just include a sample that does something like:
`refresh_interval=timedelta(days=2).total_seconds`